ui/src/pages/MyBoughtAuctionItemsPage/components/ItemListItem.tsx

```diff
@@ -0,0 +1,34 @@
+import Card from '@mui/material/Card';
+import CardMedia from '@mui/material/CardMedia';
+import { Typography } from '@material-ui/core';
+
+function ItemListItem({ auctionItem: auctionItem} : { auctionItem: any}) 
+{
+  return (
+    <Card className="MuiBox-root grid grid-cols-6 items-center gap-4">
+      <CardMedia
+        component="img"
+        image={auctionItem.photos[0]?.photoUrl}
+        alt="Your Image"
+        className="w-1/3 h-32 object-cover"
+      />
+      <CardMedia
+        component="img"
+        image={auctionItem.photos[1]?.photoUrl}
+        alt="Your Image"
+        className="w-1/3 h-32 object-cover"
+      />
+      <CardMedia
+        component="img"
+        image={auctionItem.photos[2]?.photoUrl}
+        alt="Your Image"
+        className="w-1/3 h-32 object-cover"
+      />
+        <Typography variant="body1" className="text-lg font-semibold">{auctionItem.name}</Typography>
+        <Typography variant="body1">Spent money: {auctionItem.actualPrice}</Typography>
+        <Typography variant="body1">{auctionItem.deliveryStatus ? "Delivered" : "Not delivered"}</Typography>
```

Comment:
Delivery status is enum status. Please copy enum statuses from backend and apply appropriate naming for the statuses, thanks

Reply:
Done